backend/main/src/Group/Repositories/GroupRepository.ts

```diff
@@ -79,11 +79,24 @@ export class GroupRepository implements IGroupRepository {
 	 * @param Group
 	 */
 	async update(Group: Group): Promise<undefined> {
-		await this.nishikiDbClient.saveGroup(Group.id.id, {
-			groupName: Group.name,
-			userIds: Group.userIds.map((userId) => userId.id),
-			containerIds: Group.containerIds.map((containerId) => containerId.id),
-		});
+		// delete users who are not in the updated group
+		const currentGroupUsers = await this.nishikiDbClient.listOfUsersInGroup(
+			Group.id.id,
+		);
+		const deletingUsers = currentGroupUsers.filter(
+			(user) => !Group.userIds.some((userId) => userId.id === user.userId),
+		);
+
+		await Promise.all([
+			deletingUsers.map((user) =>
+				this.nishikiDbClient.deleteUserFromGroup(Group.id.id, user.userId),
+			),
```

Comment:
I'm not sure, don't need it to spread it?

Reply:
Sorry, I don't get it. But at least I ensured that it worked in aws.